server/views/pages/goal/update/index.njk

```diff
@@ -152,13 +152,31 @@
                 ],
                 errorMessage: errors | findError('steps[' + (loop.index-1) + '][targetDateRange]')
               }) }}
+
+              <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-6">
```

Comment:
I've added this `hr` because the screenshots in the jira ticket (and the current prototype) have a horizontal rule between each step